functions/Get-DbaUserPermission.ps1

```diff
@@ -217,7 +217,7 @@ function Get-DbaUserPermission {
 
                 $sqlFile = Join-Path -Path $script:PSModuleRoot -ChildPath "bin\stig.sql"
                 $sql = [System.IO.File]::ReadAllText("$sqlFile")
-                $sql = $sql.Replace("<TARGETDB>", $db.Name)
+                $sql = $sql.Replace("<TARGETDB>", ('[' + $db.Name + ']'))
```

Comment:
I would prefer to just fix the syntax in the stig.sql file since we own it.

Reply:
Ok, I was not aware that this code is not third party but owned by dbatools. Give me some minutes...